UPDATING.md

```diff
@@ -183,6 +188,12 @@ session.add(admin)
 session.commit()
 ```
 
+## Airflow 1.10.1
```

Comment:
This change shouldn't be here. Probably needs a rebase on master.

Reply:
pretty sure i fixed it